kieker-extension-cassandra/.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -0,0 +1,63 @@
+eclipse.preferences.version=1
```

Comment:
Does it make sends to include eclipse settings into the repo?

Reply:
Done